Tests/GitHubRepositories.tests.ps1

```diff
@@ -829,970 +741,1060 @@ try
-            It 'Should not throw' {
-                { Disable-GitHubRepositorySecurityFix -Uri $repo.svn_url } |
-                    Should -Not -Throw
-            }
+        It 'Should not throw' {
+            { $repo | Enable-GitHubRepositorySecurityFix } |
+            Should -Not -Throw
         }
+    }
 
-        Context "When specifiying the 'URI' Parameter from the Pipeline" {
-            BeforeAll -ScriptBlock {
-                Enable-GitHubRepositorySecurityFix -Uri $repo.svn_url
-            }
+    AfterAll -ScriptBlock {
+        Remove-GitHubRepository -Uri $repo.svn_url -Force
+    }
+}
 
-            It 'Should not throw' {
-                { $repo | Disable-GitHubRepositorySecurityFix } |
-                    Should -Not -Throw
-            }
-        }
+Describe 'GitHubRepositories\Disable-GitHubRepositorySecurityFix' {
+    BeforeAll {
+        $repo = New-GitHubRepository -RepositoryName ([Guid]::NewGuid().Guid)
+        Enable-GitHubRepositoryVulnerabilityAlert -Uri $repo.svn_url
+        Enable-GitHubRepositorySecurityFix -Uri  $repo.svn_url
+    }
 
-        AfterAll -ScriptBlock {
-            Remove-GitHubRepository -Uri $repo.svn_url -Force
+    Context 'When Disabling GitHub Repository Security Fixes' {
+        It 'Should not throw' {
+            { Disable-GitHubRepositorySecurityFix -Uri $repo.svn_url } |
+            Should -Not -Throw
         }
     }
 
-    Describe 'GitHubRepositories\Get-GitHubRepositoryActionsPermission' {
-        BeforeAll {
-            $repoName = [Guid]::NewGuid().Guid
-            $repo = New-GitHubRepository -RepositoryName $repoName
+    Context "When specifiying the 'URI' Parameter from the Pipeline" {
+        BeforeAll -ScriptBlock {
+            Enable-GitHubRepositorySecurityFix -Uri $repo.svn_url
+        }
 
-            $allowedActions = 'All', 'LocalOnly', 'Selected', 'Disabled'
+        It 'Should not throw' {
+            { $repo | Disable-GitHubRepositorySecurityFix } |
+            Should -Not -Throw
         }
+    }
 
-        foreach ($allowedAction in $allowedActions)
-        {
-            Context "When the AllowedAction is $allowedAction" {
-                BeforeAll {
-                    $setGitHubRepositoryActionsPermissionParms = @{
-                        Uri = $repo.svn_url
-                        AllowedActions = $allowedAction
-                    }
+    AfterAll -ScriptBlock {
+        Remove-GitHubRepository -Uri $repo.svn_url -Force
+    }
+}
 
-                    Set-GitHubRepositoryActionsPermission @setGitHubRepositoryActionsPermissionParms
+Describe 'GitHubRepositories\Get-GitHubRepositoryActionsPermission' {
+    BeforeAll {
+        $repoName = [Guid]::NewGuid().Guid
+        $repo = New-GitHubRepository -RepositoryName $repoName
 
-                    $permissions = Get-GitHubRepositoryActionsPermission -Uri $repo.svn_url
-                }
+        $allowedActions = 'All', 'LocalOnly', 'Selected', 'Disabled'
```

Comment:
```suggestion
```

Looks like we either drop that, or reference it at 1391 (I think dropping it is fine).